packages/sanes-chrome-extension/src/extension/backgroundscript.ts

```diff
@@ -3,6 +3,10 @@ import { createBrowserDb } from '../logic/db';
 import { Request, RequestHandler } from './background/actions/createPersona/requestHandler';
 import { handleExternalMessage } from './background/handlers/externalHandler';
 import { internalHandler } from './background/handlers/internalHandler';
+import { updateExtensionBadge } from './background/actions/createPersona/requestExtensionBadge';
+
+//Reset extension badge to clear one after browser start because of queue reset
+updateExtensionBadge(0);
```

Comment:
I do not really understand why the CI fails, let's wait for #230 to be merged, it should work fine.

About the solution implemented in this PR LGTM.